api/proto/teleport/legacy/types/types.proto

```diff
@@ -4210,6 +4210,12 @@ message LockSpecV2 {
     (gogoproto.nullable) = true,
     (gogoproto.jsontag) = "expires,omitempty"
   ];
+  google.protobuf.Timestamp CreatedOn = 4 [
```

Comment:
I came here from your status update, just a small nit: `CreatedOn` is going to include both date and time, right? In that case, what do you think about naming it `CreatedAt` instead?

I grepped the project and it seems that we tend to use `CreatedAt` over `CreatedOn`. But tbh the only reason I comment is my Rails background where it's a convention that db fields which store datetimes are `at` and those that store dates only are `on`. ;) It seems like a useful distinction to follow though!

Reply:
Sure np, I'll convert it to be `CreatedAt`.